src/main/java/core/basesyntax/service/impl/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReaderImpl implements CsvFileReader {
```

Comment:
Code doesn't have any CSV specific logic
```suggestion
public class FileReaderImpl implements FileReader {
```